src/Workspaces/Core/Portable/FindSymbols/SymbolTree/SymbolTreeInfo_Metadata.cs

```diff
@@ -295,9 +302,12 @@ internal SymbolTreeInfo Create()
                     }
                 }
 
-                var unsortedNodes = GenerateUnsortedNodes();
+                var simpleMap = new MultiDictionary<string, ExtensionMethodInfo>();
+                var complexBuilder = ArrayBuilder<ExtensionMethodInfo>.GetInstance();
```

Comment:
you're using ArrayBuilder.GetInstance and Getinstance(var builder) inconsistently.  it's making the review harder :)

Reply:
I use different one based on how complex the containing method is, will keep it as is